Tasks/KubernetesManifestV0/src/utils/DeploymentHelper.ts

```diff
@@ -65,6 +66,9 @@ function checkManifestStability(kubectl: Kubectl, resourceTypes: Resource[]) {
         if (models.recognizedWorkloadTypesWithRolloutStatus.indexOf(resource.type.toLowerCase()) >= 0) {
             rolloutStatusResults.push(kubectl.checkRolloutStatus(resource.type, resource.name));
         }
+        if (isEqual(resource.type, constants.KubernetesWorkload.Pod, StringComparer.OrdinalIgnoreCase)) {
```

Comment:
Surround it with Try/catch ? so that we don't fail if some exception occurs while polling pod status ?

Reply:
Makes sense